src/routes/index.ts

```diff
@@ -292,4 +293,23 @@ router.route(config.UPDATE_CHECK_YOUR_ANSWERS_URL)
   .get(updateCheckYourAnswers.get)
   .post(updateCheckYourAnswers.post);
 
+// TODO: Figure out how to use router.route() instead of seperate get and post methods
+router.route(config.UPDATE_BENEFICIAL_OWNER_OTHER_URL)
+  .all(
+    authentication,
+    navigation.hasBeneficialOwnersStatement
+  )
+  .get(updateBeneficialOwnerOther.get)
+  .get() // TODO: getByID
+  .post(...validator.beneficialOwnerIndividual, checkValidations, updateBeneficialOwnerOther.post)
+  .post() // TODO: update?
+  .get(); // TODO: remove?
+
+// Copied from register routes, some of the params are from register journey so need to be updated
```

Comment:
please set to draft PR until these are sorted

Reply:
It has always been in draft